### PR TITLE
Add Home and About responsive pages

### PR DESCRIPTION
**Pull request by dennis-every**
For this PR, completed the following tasks:
1. Add index.html, index.css, and app.js files
2. Add GitHub actions workflow, gitignore, and linters: webhint, stylelint, and eslint
3. Add Bootstrap CSS and Lato font family
4. Add the navbar and font awesome icons to the home page
5. Add the hero section, add position fixed to the navbar of the home page
6. Add the program section to the home page
7. Add the speakers' section to the home page
8. Add about.html, about.css, and about hero section
9. Add the logo section to the about page
10. Add the previous summits section to the about page
11. Add the partners' section to the about page
12. Add the footer section to the about page
13. Add navbanner to the home page for desktop view
14. Add a desktop view for the navbar
15. Add a desktop view for the hero section of the home page
16. Add a desktop view for the program section of the home page
17. Add a desktop view for the speakers' section of the home page
18. Add a desktop view for the partners' section of the home page
19. Add a desktop view for the footer of the home page
20. Extract common CSS to app.css
21. Add a desktop view for navbanner to the about page
22. Add a desktop view of the navbar to the about page
23. Add a desktop view for the hero section of the about page
24. Add a desktop view for the logo section of the about page
25. Add a desktop view for the previous section of the about page
26. Add a desktop view for the partners' section of the about page
27. Add a desktop view for the footer section of the about page
28. Add a href links
29. Add JS mobile menu link handler
30. Load speaker data with JS
31. Update the hero header text with a background image over the text
32. Update the readme and MIT license files
33. Code review changes: apply appropriate margins to the speaker section, fix redundant use of h1 tags
34. Add an updated video of the project highlighting the code of select features: https://www.loom.com/share/88fe517e86ed4804abaa4fcc3e2a2ff7

### CODE REVIEW

File: assets/stylesheets/index.css [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+#speakers .speaker-card {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 12px;
+}
+
+#speakers ul .desktop-only {
+  display: none;
+}
```
Comment (Qoosim) [inline assets/stylesheets/index.css]: - [ ] It is required to produce a close replica of the guiding template. Your project looks awesome, but you need to look into the margin left  and right of the speaker section compare to what we have on the guiding design. You should also try to look into the space between each speaker picture and the text next it. At the moment, the space on your design is too wide compare to what we have on the guiding template. Kindly compare and adjust your design.

#### Your Design
your-design-speaker-desktop

### Guiding Template
temp-speaker-section-desktop

Comment (Qoosim) [reply assets/stylesheets/index.css]: - [ ] For the mobile version of the same speaker section, there is margin-bottom for each of the speaker for the readability purpose. But there is no such a margin on your design for that particular section which makes defeat the readability feature for that section. Kindly compare your design with the guiding template and work your project along side the guiding template.

### Your Design
your-speaker-mobile

### Guiding template
temp-speaker-mobile

- [ ] I would also recommend you to add margin left and right for it to look like the original design

File: about.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </div>
+    </section>
+
+    <section id="about-hero">
+      <header id="about-hero-header">
+        <h3 class="mobile-only">"Hello World!"</h3>
+        <h1>Aruba Ruby</h1>
+        <h1>Global summit 2023</h1>
```
Comment (Qoosim) [inline about.html]: - [ ] To follow html best practice, there should be only one `h1` tag in each page of any website. You can use `<br />` to divide your main title or another approach. Kindly remove one `h1` tag and find a way to make it look the same without repeating the same `h1` tag twice.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        <div id="date-venue">
+          <h3 id="date">2023.01.04(THU) ~ 05(FRI)</h3>
+          <h5 id="venue">@Cas di Cultura, Oranjestad, Aruba</h5>
+        </div>
+      </article>
+    </section>
+
+    <section id="program">
+      <header>
+        <h1>Main Program</h1>
```
Comment (Qoosim) [inline index.html]: - [ ] It is html best practice to have only one main title in a webpage which you already have one up. Kindly use any of the remaining heading `hx (2, 3, 4, 5, 6).`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+        <a href="#">See the whole the program</a>
+      </article>
+      <button type="button" class="mobile-only">
+        Join Aruba Ruby on Rails Global Summit 2023
+      </button>
+    </section>
+
+    <section id="speakers">
+      <header>
+        <h1>Featured Speakers</h1>
```
Comment (Qoosim) [inline index.html]: - [ ] It is html best practice to have only one main title in a webpage which you already have one up. Kindly use any of the remaining heading `hx (2, 3, 4, 5, 6).`

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @dennis-every, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

#  Top Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-  Awesome UI. :heavy_check_mark: 




## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

- [ ] You've done great job in your video presentation, but one thing  is missing in that presentation. You're required to highlight a piece of the code that you find interesting or  proud of making. Kindly include this in your presentation. Great job man.

**Review by ichala — APPROVED** [review]
Hi @dennis-every   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**